src/append/rolling_file/policy/compound/trigger/time.rs

```diff
@@ -484,13 +523,22 @@ mod test {
     }
 
     #[test]
-    fn pre_process() {
-        let config = TimeTriggerConfig {
-            interval: TimeTriggerInterval::Minute(2),
-            modulate: true,
-            max_random_delay: 0,
+    fn trigger_large_interval() {
+        let interval = TimeTriggerInterval::Second(i64::MAX);
+        let file = tempfile::tempdir().unwrap();
+        let logfile = LogFile {
+            writer: &mut None,
+            path: file.path(),
+            len: 0,
         };
-        let trigger = TimeTrigger::new(config);
+
+        let trigger = TimeTrigger::new(interval, false, 0);
+        trigger.trigger(&logfile).unwrap_err();
```

Comment:
Can we check the error more than just making sure it occurred? Maybe we add more error kinds in the future